### PR TITLE
Meet open source community

### LINKED ISSUE #53: Coding: Design The Meet Our Open Source Community Part Of About Page
## Description
As a developer,
I need the following mock-up to be implemented,
so that I can add as Open-Source Community Component [here](https://github.com/anitab-org/anitab-org.github.io/tree/develop/src/Components).

## Mocks
(Only the section in the red box to be developed)
about-us-part2

Font Size for heading MEET OUR OPEN SOURCE COMMUNITY: Header Medium(32pt) weight- Medium
Font Size for the description below: body 18pt weight- Light
Font Style- Avenir
Font Color (heading): #103B81
Font Color (Body): #103B81

Margin between heading and description: 16pt
Image size: 81px X 81px
horizontal and vertical gutter between two images: 2px 
corner radius: 8px
Top margin of the entire image component on page: 1320px
image block size: height:310px ,width:1638px

[Design Development Link](https://xd.adobe.com/view/ba1323d0-760e-4f29-4658-c83848c894ba-cc22/)

### Update [Required]
- [ ] Instead of dummy images in MEET OUR OPEN SOURCE COMMUNITY, get GitHub profile images of all members and arrange them according to mock-up shared, also when a user hovers on a particular image redirect them to respective Github profiles.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

### PR DESCRIPTION
**Pull request by parthsarthiprasad**
### Description
Fixes #53
OSC

### Type of Change:

- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by parthsarthiprasad** [comment]
@nandini45 @Sonal240 Please review it, Thanks.

**Review by Sonal240 — APPROVED** [review]
LGTM good to see that you are making use of api and not hard coding ....but still @nandini45 @annabauza please do review it

**Review by nandini45 — APPROVED** [review]
yeah i like your work its good to go but a final review from Anna is mandatory. we will merge it asap, thanks for the contribution
@parthsarthiprasad

File: src/Components/AboutUs/Oscommunity.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        flexDirection: "row",
+        margin: 32,
+        alignContent: "space-around"
+    },
+    container: {
+        padding: 1,
+        marginTop: 8,
+    },
+    image: {
+      width: 81,
```
Comment (parthsarthiprasad) [inline src/Components/AboutUs/Oscommunity.js:57]: @annabauza while reviewing, I tried to follow specifications where they mentioned #53  each image of 81x81, which looks quite large, the gif is made on 24x24 image resolution.(width x height). shall I decrease this appropriately? or stick to specification.

Comment (nandini45) [reply src/Components/AboutUs/Oscommunity.js:57]: can you give me some time i will try and test it by saturday, and tell you further??

Comment (parthsarthiprasad) [reply src/Components/AboutUs/Oscommunity.js:57]: sure works fine.

**Comment by parthsarthiprasad** [comment]
@nandini45 can `hacktoberfest-accepted` label be added to this PR if acceptable. thanks. :smile: :handshake:

**Review by annabauza — APPROVED** [review]

**Pull request merged**